feature-libs/checkout/base/components/checkout-payment-method/checkout-payment-form/checkout-payment-form.component.ts

```diff
@@ -201,22 +233,57 @@ export class CheckoutPaymentFormComponent implements OnInit {
     this.sameAsDeliveryAddress = !this.sameAsDeliveryAddress;
   }
 
-  getAddressCardContent(address: Address): Card {
-    let region = '';
-    if (address.region && address.region.isocode) {
-      region = address.region.isocode + ', ';
+  /**
+   * TODO: (#CXSPA-53) Remove feature returnAsync in 6.0
```

Comment:
This comment seems to indicate that we want to remove the return of an observable in 6.0 which is not what we want. Please ellaborate on this comment.